packages/lesmis-server/src/utils/verifyLoginAccess.ts

```diff
@@ -7,8 +7,8 @@ import { AccessPolicy } from '@tupaia/access-policy';
 import { PermissionsError } from '@tupaia/utils';
 import { LESMIS_PERMISSION_GROUP } from '../constants';
 
-export const verifyLoginAccess = (accessPolicy: AccessPolicy) => {
-  const hasAccess = new AccessPolicy(accessPolicy.policy).allowsAnywhere(LESMIS_PERMISSION_GROUP);
+export const verifyLoginAccess = (accessPolicy: any) => {
```

Comment:
We should try to be explicit with types where we can. I'm not sure if this is a `string` or a POJO (which we generally use `Record<string, any>` for), but either way it would be worth marking as such